src/hooks/use-breakpoints-google.tsx

```diff
@@ -0,0 +1,18 @@
+import useMediaQuery from '@mui/material/useMediaQuery'
+
+const useBreakpointsGoogle = () => {
```

Comment:
do you really need this hook, as I see, you can use our useBreakpoints for that

Reply:
fixed